Android/app/src/main/java/fashiome/android/activities/ProductFormActivity.java

```diff
@@ -380,7 +380,7 @@ public void onConnected(Bundle dataBundle) {
         if (location != null) {
             Toast.makeText(this, "GPS location was found! Lat:"+ location.getLatitude() +" Long:"+location.getLongitude(), Toast.LENGTH_SHORT).show();
             LatLng latLng = new LatLng(location.getLatitude(), location.getLongitude());
-            product.setAddress(new Address(latLng));
+            product.setAddress(new Address(Item.getRandomLocation(latLng.latitude, latLng.longitude,5000)));
```

Comment:
Thanks,
Rakhee

On Mar 19, 2016, at 3:01 PM, Deborshi <notifications@github.com<mailto:notifications@github.com>> wrote:


In Android/app/src/main/java/fashiome/android/activities/ProductFormActivity.java<https://github.com/deborshisaha/ModaRenta/pull/57#discussion_r56755801>:

> @@ -380,7 +380,7 @@ public void onConnected(Bundle dataBundle) {
>          if (location != null) {
>              Toast.makeText(this, "GPS location was found! Lat:"+ location.getLatitude() +" Long:"+location.getLongitude(), Toast.LENGTH_SHORT).show();
>              LatLng latLng = new LatLng(location.getLatitude(), location.getLongitude());
> -            product.setAddress(new Address(latLng));
> +            product.setAddress(new Address(Item.getRandomLocation(latLng.latitude, latLng.longitude,5000)));


@rakheegupta<https://github.com/rakheegupta>

You have your trial code in there, then you have code that you are using to debug your own stuff and then you are merging it directly to master. We are really upset with the way you are approaching things here.

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/deborshisaha/ModaRenta/pull/57/files/b5a0e76ed0e25c34f4b9737bf76df95ed8676ffb#r56755801>